### PR TITLE
Support node16

### PR DESCRIPTION
**Pull request by barbara79**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -16,14 +16,21 @@ aliases:
...
 executors:
-  node_14:
+  docker:
     docker:
-      - image: cimg/node:14.19.3
+      - image: cimg/node:14.20.0
 
 version: 2.1
+orbs:
+  node: circleci/node@5.0.2
```
Comment (ajimae) [inline .circleci/config.yml:25]: Do we really need this?

Comment (barbara79) [reply .circleci/config.yml:25]: yes, otherwise the build will fail I found this https://circleci.com/developer/orbs/orb/circleci/node

Comment (barbara79) [reply .circleci/config.yml:25]: I tried without but the built for each version of node was failing

Comment (ajimae) [reply .circleci/config.yml:25]: Well since you are using circleci orbs, this can pass.

**Pull request merged**